### ISSUE TITLE
UnicodeString alignment

### ISSUE THREAD

**Issue by wolfcod** [qilingframework/qiling#795]
***Describe the bug**
UnicodeString object is not aligned to UNICODE_STRING data structure on Windows x64

**Sample Code**
```windbg x86
0:000> dt _UNICODE_STRING
ntdll!_UNICODE_STRING
   +0x000 Length           : Uint2B
   +0x002 MaximumLength    : Uint2B
   +0x004 Buffer           : Ptr32 Wchar
```

```windbg x64
0:000> dt ntdll!_UNICODE_STRING
   +0x000 Length           : Uint2B
   +0x002 MaximumLength    : Uint2B
   +0x008 Buffer           : Ptr64 Wchar
```

**Expected behavior**
Using UnicodeString.Buffer we except the right pointer to string in memory. In a 64-bit environment, the pointer is invalid and there is an exception using ql.mem.read

**Additional context**
The problem is present using the subsystem x64 but not on 32-bit environment.

**Comment by xwings** [comment]
I guess is will not be a hard fix. A PR from you will be really cool.

**Comment by wtdcode** [comment]
How to reproduce it?

**Comment by wolfcod** [comment]
```cpp
NTSYSAPI VOID RtlInitUnicodeString(
  PUNICODE_STRING         DestinationString,
  __drv_aliasesMem PCWSTR SourceString
);```

`ntdll!RtlInitUnicodeString` amd64
`mov     [rcx], r8
mov     [rcx+8], rdx`

This is the implementation on ntdll of this api. As described in PR, this api is hooked only on NTOSKRNL, but not on userland api.

**Comment by xwings** [comment]
merged #796
